packages/orbit-components/src/CountryFlag/README.md

```diff
@@ -16,18 +16,18 @@ After adding import into your project you can use it simply like:
 
 Table below contains all types of the props available in CountryFlag component.
 
-| Name     | Type                                             | Default       | Description                          |
-| :------- | :----------------------------------------------- | :------------ | :----------------------------------- |
-| code     | [`enum`](#enum)                                  | `"undefined"` | Code for the displayed country flag. |
-| dataTest | `string`                                         |               | Optional prop for testing purposes.  |
-| id       | `string`                                         |               | Set `id` for `CountryFlag`           |
-| name     | `string`                                         |               | The name for the flag.               |
-| role     | `React.HTMLAttributes<HTMLImageElement>["role"]` | `"img"`       | ARIA role for the flag image.        |
-| size     | [`enum`](#enum)                                  | `"medium"`    | The size of the CountryFlag.         |
+| Name     | Type                                             | Default       | Description                                |
+| :------- | :----------------------------------------------- | :------------ | :----------------------------------------- |
+| code     | [`enum`](#enum)                                  | `"undefined"` | Code for the displayed country flag.       |
+| dataTest | `string`                                         |               | Optional prop for testing purposes.        |
+| id       | `string`                                         |               | Set `id` for `CountryFlag`.                |
+| name     | `string`                                         | `""`          | Defines the `alt` attribute for the image. |
```

Comment:
you wrote that `""` is a default value for for `name`, but there is no default value for `name` set in `packages/orbit-components/src/CountryFlag/index.tsx`